src/util/cable_common.F90

```diff
@@ -27,6 +27,10 @@ MODULE cable_common_module
 USE cable_runtime_opts_mod ,ONLY : satuparam
 USE cable_runtime_opts_mod ,ONLY : wiltparam
 
+! two imports below inserted by rk4417 - phase2
+USE cable_pft_params_mod, ONLY : vegin
+USE cable_soil_params_mod, ONLY : soilin
```

Comment:
Why? They don't seem to be used here?

Reply:
These appear to go back to a time (phase 1 and before) when at least vegin was used inside cable_common. Specifically to the time before subroutine init_veg_from_vegin was moved from cable_common.F90 to cable_parameters.F90. I deleted both lines